frontend/src/pages/meditation/meditation-entries-list/meditation-entry/meditation-entry.tsx

```diff
@@ -0,0 +1,33 @@
+import meditationListPlaceholder from '#assets/img/meditation-list-placeholder.jpg';
+import { Icon, Link } from '#libs/components/components.js';
+import { IconColor } from '#libs/enums/enums.js';
+
+import { type Entry } from '../../libs/types/types.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  track: Entry;
+};
+
+const MeditationEntry: React.FC<Properties> = ({ track }) => {
+  return (
+    <div className={styles['track']}>
+      <img
+        src={meditationListPlaceholder}
+        alt="Meditation track background"
```

Comment:
```suggestion
        alt="Meditation track image"
```

Reply:
Done

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111736110/9310237f-ae1b-4375-880b-0723c9ea1099)
I have this error, so would name it just "Meditation entry"